aurget

```diff
@@ -331,7 +342,7 @@ set_defaults() {
   silent=false
   sort_mode='name'
   sync_mode='install'
-  temp_directory='/tmp/aurget'
+  pkgbuild_directory='~/.cache/aurget'
```

Comment:
- I would call this `cache_directory`
- It should follow XDG, `cache_directory=${XDG_CACHE_HOME:-$HOME/.cache}/aurget`